wandb/sdk/verify/verify.py

```diff
@@ -136,16 +150,16 @@ def check_run(api: Api) -> None:
             )
 
         wandb.save(filepath)
-
     public_api = wandb.Api()
-    try:
-        prev_run = public_api.run("{}/{}/{}".format(entity, PROJECT_NAME, run_id))
-    except Exception:
+    prev_run = retry_fn(
```

Comment:
This doesnt need to be retried it is already retried in RetryingClient in the public_api

I think (though i never looked at the logs of thee failure) that the part that needed to be retried is querying the files table later on in this function.

The backend currently does not guarantee the files table is up to date at the time of wandb.finish()